### PR TITLE
#6 fixing pheval post processing breaking changes

### PR DESCRIPTION
**Pull request by yaseminbridges**
The `score_order` (either `ascending` or `descending`) must now be specified in the config.yaml and has been specified when calling PhEval post-processing methods. This fixes the breaking change in the PhEval post-processing methods once merged in

### CODE REVIEW

File: src/pheval_exomiser/config_parser.py [hunk 1]
```diff
@@ -5,113 +5,6 @@
...
-#     scramble_phenopacket: float
-#
-#
-# @serde
-# @dataclass
-# class ExomiserPrepareUpdatePhenopacket:
-#     update: bool
-#     gene_identifer_to_update: str
-#
-#
```
Comment (julesjacobsen) [inline src/pheval_exomiser/config_parser.py:21]: Where did these all go? Are they part of the generic pheval setup now?

**Review by julesjacobsen — COMMENTED** [review]
Did all the exomiser_config_parser.py functionality get delegated to the main pheval runner?

File: src/pheval_exomiser/config_parser.py (hunk 1 above)
Comment (yaseminbridges) [reply src/pheval_exomiser/config_parser.py:21]: This was all part of the first config schema I had thought of which included the "prepare" steps of updating Phenopacket gene identifiers, scrambling phenopackets and spiking the VCFs. I removed these from the config as this is part of the MakeFile now. The functionality for reading the exomiser runner config is located in the `config_parser.py` in this repo as the config for each plugin is likely to be different

Comment (julesjacobsen) [reply src/pheval_exomiser/config_parser.py:21]: I though so - was just double-checking!

**Pull request merged**